packages/graphql/src/scripts/2025-01-15_cleanup_solutions_feedbacks_elements.ts

```
@@ -0,0 +1,348 @@
+            //       hasSampleSolution: false,
+            //       hasAnswerFeedbacks: false,
+            //       choices: element.options.choices.map((choice) => ({
+            //         ...choice,
+            //         correct: undefined,
+            //         feedback: undefined,
+            //       })),
+            //     },
+            //   },
+            // })
+
+            missingChoicesSampleSolution++
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)